.env

```diff
@@ -0,0 +1,4 @@
+source venv/bin/activate
```

Comment:
why would you want to activate your virtualenv from the .env file?

Reply:
So that autoenv kicks in and activates the virtualenv automatically every time i cd into my project

Reply:
If it works using autoenv, then that's good stuff